cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperator.java

```diff
@@ -1098,8 +1100,9 @@ public KafkaAssemblyOperator(Vertx vertx, boolean isOpenShift,
                                             configMapOperations.get(kafkaAssembly.getMetadata().getNamespace(), ((ExternalLogging) userOperator.getLogging()).getName()) :
                                             null) : null;
 
+                            Map<String, String> annotations = new HashMap<>();
                             this.entityOperator = entityOperator;
-                            this.eoDeployment = entityOperator.generateDeployment(isOpenShift);
+                            this.eoDeployment = entityOperator.generateDeployment(annotations, isOpenShift);
```

Comment:
do we need passing this `annotations` map if it's empty? I don't see the need anymore, right? The `generateDeployment` already pass an empty map to the `createDeployment` from the base class.